.travis.yml

```diff
@@ -12,12 +12,12 @@ matrix:
     - php: 5.4
     - php: 5.5
     - php: 5.6
-    - php: 5.6
-      env: COMPOSER_FLAGS='--prefer-lowest --prefer-stable'
     - php: hhvm
     - php: nightly
     - php: 7.0
       env: COVERAGE=yes
+    - php: 7.0
+      env: COMPOSER_FLAGS='--prefer-lowest --prefer-stable'
```

Comment:
These are two different jobs. One with the current dependencies (on PHP 7), and another with the lowest one.